### PR TITLE
small adjustment

### PR DESCRIPTION
**Pull request by jmariomejiap**

### CODE REVIEW

**Review by pranavmalaviya — APPROVED** [review]
Looks good @jmariomejiap 🚀

File: vmware/README.md [hunk 1]
```diff
@@ -31,7 +33,13 @@
 
 3. (Optional) If you want to follow the build process in GUI, then you need to turn the `headless` bool to `false` in the `ubuntu-18-04-amd64.json` file.
 
-4. Build the packer template by running one of these commands:
+4. (Optional) If you have built a VM appliance before, You will have a folder with its content inside the `/builds/` directory. You will need to delete any existing build before proceeding. Run this command to do so:
```
Comment (prees1) [inline vmware/README.md:36]: not sure if this is useful or not https://www.packer.io/docs/commands/build#force

Comment (jmariomejiap) [reply vmware/README.md:36]: Oh yeah, that would be great. I'll try it out. Thanks @prees1

Comment (pranavmalaviya) [reply vmware/README.md:36]: Oh nice!
> The exact behavior of a forced build is left to the builder.
The next time I'm building with VMware, I would like to check if it persists the downloaded ISO image in the build directory or completely removes it. (The argument is only for cases where the user doesn't have the image in the `iso` folder.)

**Comment by SamDesmondKing** [comment]
@pranavmalaviya Maybe we can merge or close this?

**Comment by pranavmalaviya** [comment]
@SamDesmondKing Yeah, I'll resolve the conflict, edit the docs as per Phil's comment and merge it tomorrow.

File: vmware/ubuntu-18-04/README.md [hunk 2]
```diff
@@ -12,50 +12,41 @@
...
-
-1. Navigate to the image folder you plan to generate. For example, ubuntu-18-04
-
-    ```sh
-    cd ubuntu-18-04
-    ```
-
-2. (Optional) Add following env vars to export the logs in a file
-   - For Unix/Mac:
+## List of software installed on Ubuntu OVA
```
Comment (SamDesmondKing) [inline vmware/ubuntu-18-04/README.md:17]: I think this header is no longer used?

**Review by SamDesmondKing — APPROVED** [review]
Looks good, just one formatting question.

File: vmware/ubuntu-18-04/README.md (hunk 2 above)
Comment (pranavmalaviya) [reply vmware/ubuntu-18-04/README.md:17]: Good catch. Missed that one in the git conflicts. Added installed software section for both, Ubuntu and Windows, applicances.

**Pull request merged**